src/platform_impl/windows/event_loop/runner.rs

```diff
@@ -62,6 +62,7 @@ pub(crate) enum Event {
     Window { window_id: WindowId, event: WindowEvent },
     BufferedScaleFactorChanged(HWND, f64, PhysicalSize<u32>),
     // FIXME(madsmtm): Coalesce these into a flag (or similar) instead of handling them as events.
+    // https://github.com/rust-windowing/winit/pull/3687
```

Comment:
it's better to link the issue itself, but given that we have PR around, we probably should just drop the link all together?

Reply:
Problem was that there was no issue for it. I think it makes sense to leave the link in, and then just remove it in the new PR I've opened (https://github.com/rust-windowing/winit/pull/4150).

Reply:
Hm, I've assumed there certainly was... I had a memory that there was somewhere an issue about this on Windows and that proposal was to let the user know about failure.